WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java

```diff
@@ -150,6 +150,16 @@ void removePostInfoFromForegroundNotification(@NonNull PostModel post, @Nullable
     void removeMediaInfoFromForegroundNotification(@NonNull List<MediaModel> mediaList) {
         if (sNotificationData.totalMediaItems >= mediaList.size()) {
             sNotificationData.totalMediaItems -= mediaList.size();
+            // update Notification now
+            updateForegroundNotification(null);
+        }
+    }
+
+    void removeOneMediaItemInfoFromForegroundNotification() {
+        if (sNotificationData.totalMediaItems >= 1) {
```

Comment:
Should this be `>= 0` instead? Also, minor nit: there's an extra space before ` --`.

Reply:
no, if it's 0 we can't take 1 from it (that would make it a negative count). We could express it like `> 0` or `>=1` which are equivalent.
Removed the extra space in 10a869a

Reply:
Apparently I took my brain out before I made that comment. Ignore me!